lib/foreman_custom_tab/engine.rb

```diff
@@ -11,22 +11,19 @@ class Engine < ::Rails::Engine
     config.autoload_paths += Dir["#{config.root}/app/overrides"]
   
     
-    initializer 'foreman_custom_tab.configure_assets', :group => :assets do
-      SETTINGS[:foreman_custom_tab] = { assets: { precompile: assets_to_precompile } }
-    end
+    #initializer 'foreman_custom_tab.configure_assets', :group => :assets do
```

Comment:
lets just remove this rather than commenting it. Git history will provide the code if it is needed again. But I doubt we will ever need this code.

Reply:
sure